Gordon360/Web.config

```diff
@@ -13,22 +13,19 @@
     <add key="webpages:Enabled" value="false" />
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-
     <!-- @Production only
     <add key="DEFAULT_ACTIVITY_IMAGE_PATH" value="https://360api.gordon.edu/browseable/uploads/Default/activityImage.png" />
     <add key="DEFAULT_PROFILE_IMAGE_PATH" value="https://360api.gordon.edu/browseable/profile/Default/profile.png" />
     <add key="DEFAULT_PREF_IMAGE_PATH" value="\\go\pref_photos\" />
     <add key="DEFAULT_IMAGE_PATH" value="\\go\photos\" />
     <add key="DEFAULT_ID_SUBMISSION_PATH" value="\\go\ID_photo_submissions\" />
     -->
```

Comment:
We will need to update master once this is merged so that it points to production files